package.json

```diff
@@ -41,6 +41,8 @@
     "webpack-concat-plugin": "^3.0.0"
   },
   "devDependencies": {
+    "jasmine-browser-runner": "^0.9.0",
```

Comment:
To make the this change more pragmatic and readable, I would suggest we set jasmine as a test script in our package.json (also inline with Jasmine's documentation)
`"scripts": { "test:jasmine": "jasmine-browser-runner runSpecs" }`

Reply:
Two scripts added, `test:jasmine` which will run the test in a browser and `test:jasmine-headless` for running in CLI